### PR TITLE
Re-add PHPStan to CI

### PR DESCRIPTION
**Pull request by szepeviktor**
All 🍏

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -130,3 +130,35 @@ jobs:
...
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+      - name: Cache composer dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
+          restore-keys: ${{ runner.os }}-composer-
+      - name: Install Composer dependencies
+        run: composer update --no-progress --prefer-dist --optimize-autoloader
+      - name: Run PHPCS checks
```
Comment (mundschenk-at) [inline .github/workflows/ci.yml]: Should be "PHPStan"

File: phpstan.neon [hunk 2]
```diff
@@ -31,8 +31,7 @@ parameters:
     ignoreErrors:
         # Uses func_get_args()
         - '#^Function apply_filters(_ref_array)? invoked with [34567] parameters, 2 required\.$#'
-        - '#^Parameter \#[1-9] \$[a-z_]+ of function [a-z_]+ expects GdImage, GdImage\|resource given\.$#'
```
Comment (mundschenk-at) [inline phpstan.neon:34]: This line should not be removed (eventually, development will happen on PHP 8).

File: .github/workflows/ci.yml (hunk 1 above)
Comment (szepeviktor) [reply .github/workflows/ci.yml]: Excuuuuuuse me!

**Comment by mundschenk-at** [comment]
Could you please rebase/squash things into one or two commits (probably one for phpstan.neon and one for the workflow)?

**Comment by szepeviktor** [comment]
Okay I learn what rebasing is.

**Comment by szepeviktor** [comment]
This is the current analysis result.

kép

May I leave it as it is?

**Comment by szepeviktor** [comment]
Done.

BTW GitHub has a squash feature on the <kbd>Merge</kbd> button.

**Comment by mundschenk-at** [comment]
> BTQ GitHub has a squash feature on the Merge button.

Yes, but that squashes EVERYTHING into a single commit by me.

**Comment by szepeviktor** [comment]
You could add `# commented out` PHP 7.4 ignores.

**Comment by mundschenk-at** [comment]
Hi @szepeviktor, do you want to refresh this patch or should I update the configuration myself?

**Pull request merged**